Consuming.md

```diff
@@ -0,0 +1,28 @@
+# Consumers 
```

Comment:
Quoting @dmihalcik-virtru , so we can have a threaded convo 😅 

> The stuff in Consuming.md should be moved into the READMEs - e.g. configuring the services should be in service/README.md and using the SDK should be in sdk/README.md

I see the value in that idea 👍 I like it as a contributor to the project.

But, I'm not sure that clarifies users versus developers.  You need to know different things to use `opentdf/platform` versus contribute to `opentdf/platform`.

Perhaps we can do both?  It looks like we already have two use-cases / personas:
* HSM: https://github.com/opentdf/platform/blob/main/README.md
* Non-HSM: https://github.com/opentdf/platform/blob/main/README-hsm.md

Couldn't we just expand this thinking to give an end-user a nice welcoming / happy path?

Reply:
It looks like the 'default' behavior will be non-HSM, so maybe separating out 'how to set up hsm' makes sense, after we fix building without an hsm library (see https://github.com/opentdf/platform/issues/565)